src/http/Request.php

```diff
@@ -184,7 +184,7 @@ public function isPatch()
      */
     public function isSecure()
     {
-        return ($this->gethttps() || $this->getServerPort() && $this->getServerPort() == '443') ? true : false;
+        return $this->gethttps() || $this->getServerPort() && $this->getServerPort() == '443';
```

Comment:
According to the official [PHP doc](https://www.php.net/manual/en/reserved.variables.server.php) about `$_SERVER['SERVER_PORT']`, the port number type is string.

For value comparing performance, it should use the `===` to make value equals strictly.

Reply:
:+1: